dependencies.yaml

```
@@ -61,7 +61,7 @@ ext-mbedtls:
 # everest-testing
 everest-utils:
   git: https://github.com/EVerest/everest-utils.git
-  git_tag: v0.1.6
+  git_tag: 53c5fdf
```

I think this should be 4a5ce95 or even better a054460 (afterwards we should tag another everest-utils version anyway)
```suggestion
  git_tag: a054460
```